<launch>

    <!-- static transform between laser and the robot -->
    <node pkg="tf" type="static_transform_publisher" name="base_2_lidar1" args="-0.49 0.2 0.22 0 0 -3.1415926535 base_link lidar1 670"/>
    <!-- static transform between laser and the robot -->
    <node pkg="tf" type="static_transform_publisher" name="base_2_lidar2" args="0.49 -0.2 0.22 0 0 0 base_link lidar2 670"/>
    
    <param name="use_sim_time" value="true"/>

    <!--Load parameters from config files-->
    <node name="lidar_fusion_node" pkg="lidar_fusion" type="lidar_fusion" output="screen">
      <rosparam command="load" file="$(find lidar_fusion)/config/config.yaml" />
    </node>

</launch>
